ConfigurationSystem/Client/VOMS2CSSynchronizer.py

```diff
@@ -139,13 +143,26 @@ def __init__(self, vo, autoModifyUsers=True, autoAddUsers=True,
     self.autoDeleteUsers = autoDeleteUsers
     self.autoLiftSuspendedStatus = autoLiftSuspendedStatus
     self.voChanged = False
+    self.syncPlugin = None
+
+    if syncPluginName:
+      objLoader = ObjectLoader()
+      _class = objLoader.loadObject(
+          'ConfigurationSystem.Client.SyncPlugins.%sSyncPlugin' %
+          syncPluginName, '%sSyncPlugin' % syncPluginName)
+
+      if not _class['OK']:
+        raise Exception(_class['Message'])
+
+      self.syncPlugin = _class['Value']()
```

Comment:
Am I correct in thinking this is initialised once per process and not once per sync run? If so, how about initialising the sync plugin once per call to `syncCSWithVOMS`? It would mean:

* the plugins can be changed without restarting the VOMS2CSAgent
* plugins can cache data/connections without worrying about invalidating this cache

Reply:
Ah sorry ignore this, it's initialised in `VOMS2CSAgent.execute`